screens/Home/HomeScreen.tsx

```diff
@@ -9,15 +9,15 @@ import React, { useState } from "react"
 import SectionTabs from "../../components/SectionTabs/SectionTabs"
 import EventScreen from "./EventScreen/EventScreen"
 import AnnouncementScreen from "./AnnouncementScreen/AnnouncementScreen"
+import { NavigationProp, ParamListBase } from "@react-navigation/native"
 
+interface HomeScreenProps {
+  navigation: NavigationProp<ParamListBase>
+}
 
-const HomeScreen = () => {
+const HomeScreen = ({ navigation }: HomeScreenProps) => {
```

Comment:
why do you wrap a props in an interface? I think that you do not need to do that. It was the problem that @alexandre-mrt had.

Reply:
I did it because when navigating with parameters, typescript gives an error without it (when calling navigation.navigate(...)). I didn't want to leave typescript errors.